src/ebfloeseg/load.py

```diff
@@ -44,13 +50,20 @@ def load(
     format: str = "image/tiff",
 ):
 
-    match kind:
-        case ImageType.truecolor:
+    match (satellite, kind):
+        case (Satellite.terra, ImageType.truecolor):
             layers = "MODIS_Terra_CorrectedReflectance_TrueColor"
-        case ImageType.cloud:
+        case (Satellite.terra, ImageType.cloud):
             layers = "MODIS_Terra_Cloud_Fraction_Day"
-        case ImageType.landmask:
+        case (Satellite.aqua, ImageType.truecolor):
+            layers = "MODIS_Aqua_CorrectedReflectance_TrueColor"
+        case (Satellite.aqua, ImageType.cloud):
+            layers = "MODIS_Aqua_Cloud_Fraction_Day"
+        case (_, ImageType.landmask):
             layers = "OSM_Land_Mask"
```

Comment:
In the spirit of using enums, what about putting these values for the layers in one as well?

Reply:
I'm not sure. Right now, I can only think of this one place where that Enum would be used, and the enum would be a 1:1 reflection of the structure of this `case`/`match` block. 

As soon as the layer names get used elsewhere in addition to here, I think making it an Enum makes total sense. But for now, I think the simplest way is to leave these as simple strings within this one function.